holoviews/core/spaces.py

```diff
@@ -881,11 +881,12 @@ class DynamicMap(HoloMap):
         If the callable is an instance of Callable it will be used
         directly, otherwise it will be automatically wrapped in one.""")
 
-    streams = param.List(default=[], constant=True, doc="""
-       List of Stream instances to associate with the DynamicMap. The
-       set of parameter values across these streams will be supplied as
-       keyword arguments to the callback when the events are received,
-       updating the streams.""" )
+    streams = param.ClassSelector(default=[], class_=(dict, list), constant=True, doc="""
+       List of Stream instances to associate with the DynamicMap or a
+       dictionary mapping parameters or panel widgets to callback
+       argument names. In the list case, the set of parameter values
+       across these streams will be supplied as keyword arguments to the
+       callback when the events are received, updating the streams.""" )
```

Comment:
This should probably explain that dicts are accepted but are then converted into the list format.

Reply:
Addressed in e4dc46149